src/uTensor.h

```diff
@@ -4,9 +4,18 @@
  * Core bits
  */
 // This one selects platform specific stuff. probably should come first
-#include "uTensor/core/uTensor_util.hpp"
+#include "uTensor/core/context.hpp"
+#include "uTensor/core/errorHandler.hpp"
+#include "uTensor/core/memoryManagementInterface.hpp"
 #include "uTensor/core/modelBase.hpp"
 #include "uTensor/core/operatorBase.hpp"
+#include "uTensor/core/quantizationPrimitives.hpp"
+#include "uTensor/core/tensor.hpp"
+#include "uTensor/core/tensorBase.hpp"
+#include "uTensor/core/TensorMap.hpp"
+#include "uTensor/core/types.hpp"
+#include "uTensor/core/utensor_string.hpp"
+#include "uTensor/core/uTensor_util.hpp"
```

Comment:
```suggestion
#include "uTensor/core/modelBase.hpp"
#include "uTensor/core/operatorBase.hpp"
#include "uTensor/core/uTensor_util.hpp"
```

Reply:
Most of these includes dont need to be delegated here since they are for internal use only and exposed-ish by other headers. You should only need to care about models, operators, and the util